guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Hammer-cli.adoc

```diff
@@ -0,0 +1,56 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Hammer_CLI_{context}"]
+= Using IdM credentials to log in to the {Project} Hammer CLI
+
+This section describes how to log in to your {Project} Hammer CLI with your {FreeIPA} (IdM) login and password.
+
+.Prerequisites
+* You have enrolled your {ProjectServer} into IdM and configured it to use IdM for authentication.
+More specifically, you have enabled access both to the {ProjectWebUI} and the {Project} API.
```

Comment:
Should line 7 and 8 be shortened to a single line? "more specifically" reads a bit off as in "the first sentence was not 100% correct". Maybe combine them?